Ports/openrct2/package.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='openrct2'
+version='0.4.3'
+auth_type='sha256'
+depends=(
+    'SDL2'
+    'curl'
+    'flac'
+    'fontconfig'
+    'freetype'
+    'libicu'
+    'libogg'
+    'libpng'
+    'libvorbis'
+    'libzip'
+    'openssl'
+    'speexdsp'
+    'zlib')
+files="https://github.com/OpenRCT2/OpenRCT2/archive/refs/tags/v${version}.tar.gz ${port}-${version}.tar.gz 1269759b3a90717f379cd375f30629e3f24d3fdeb0c54cc1dca03676e64731ef"
+useconfigure='true'
+workdir="OpenRCT2-${version}"
+configopts=(
+	"-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+	"-DCMAKE_INSTALL_PREFIX=${DESTDIR}/usr/local/"
+    "-DCMAKE_BUILD_TYPE=Release"
+	"-DDISABLE_DISCORD_RPC=ON"
+	"-DDISABLE_GOOGLE_BENCHMARK=ON"
+    "-DWITH_TESTS=OFF"
+    "-DDISABLE_OPENGL=ON")
+icon_file='resources/logo/icon_x16.png'
+launcher_name='OpenRCT2'
+launcher_category='Games'
+launcher_command='/usr/local/bin/openrct2'
+
+configure() {
+    mkdir -p "${PORT_BUILD_DIR}/OpenRCT2-${version}-build"
+    cd "${PORT_BUILD_DIR}/OpenRCT2-${version}-build"
+    cmake -G Ninja "${configopts[@]}" "${PORT_BUILD_DIR}/OpenRCT2-${version}"
+}
+
+build() {
+    prebuilt="OpenRCT2-${version}-linux-x86_64.tar.gz"
+    g2_path='OpenRCT2/data/g2.dat'
+
+    cd "${PORT_BUILD_DIR}/OpenRCT2-${version}-build"
+    curl ${curlopts:-} "https://github.com/OpenRCT2/OpenRCT2/releases/download/v${version}/${prebuilt}" -L -o "$prebuilt"
```

Comment:
This we could probably add as an additional entry in `files` instead of downloading it manually during the building phase.